lib/postgres.go

```diff
@@ -2916,3 +2944,50 @@ func (postgres *Postgres) GetNotifications(publicKey string) ([]*PGNotification,
 
 	return notifications, nil
 }
+
+//
+// Postgres Test tooling
```

Comment:
IMO this is better off in a test_utils or postgres_test file and not here.

Reply:
agreed with MF^